pkg/schemas/model.go

```diff
@@ -73,9 +73,9 @@ type Type struct {
 	Ref     string `json:"$ref,omitempty"`    // section 7
 	// RFC draft-wright-json-schema-validation-00, section 5
 	MultipleOf           int              `json:"multipleOf,omitempty"`           // section 5.1
-	Maximum              int              `json:"maximum,omitempty"`              // section 5.2
+	Maximum              float64          `json:"maximum,omitempty"`              // section 5.2
```

Comment:
From the specification itself... http://json-schema.org/draft/2019-09/json-schema-validation.html#rfc.section.6.2.2 (and same for minimum)

Reply:
btw, all section numbers are wrong

Reply:
Al the section numbers refer to the quoted draft. We should sync with the latest draft.

Reply:
I could do it, but probably in another PR. This one is focusing on the validations feature. Blocked by [this](https://github.com/atombender/go-jsonschema/pull/10#issuecomment-584063481) weird bug which I don't know how to solve/avoid